Source/GSNibLoading.m

```diff
@@ -2005,22 +2004,33 @@ proxy objects (-1 = File's Owner, -2 = First Responder,
     {
       NSInteger oid = [(id)NSMapGet(_oids, obj) intValue];
       if (oid >= 0)
-        {
-          if ([obj respondsToSelector: @selector(realObject)])
-            {
-              obj = [obj realObject];
-            }
-          if ([obj respondsToSelector: @selector(awakeFromNib)])
-            {
-              [obj awakeFromNib];
-            }
-        }
+	{
+	  if ([obj respondsToSelector: @selector(realObject)])
+	    {
+	      obj = [obj realObject];
+	    }
+
+	  // Awaken the object....
+	  [obj awakeFromNib];
+
+	  // If we are in Gorm/IB, set the object up for display
+	  if ([NSClassSwapper isInInterfaceBuilder] == YES)
+	    {
+	      [obj prepareForInterfaceBuilder];
+	    }
+	}
     }
 
   // awaken the owner
   if ([owner respondsToSelector: @selector(awakeFromNib)])
```

Comment:
Maybe this is also no longer needed?

Reply:
The call to awakeFromNib for the owner is because it's not one of the objects I archived but it still needs to have this call done.  The check is no longer needed for respondsToSelector:.

Reply:
I have updated this to remove the check.